hack/multikueue-e2e-test.sh

```diff
@@ -52,13 +52,17 @@ function startup {
             mkdir -p "$ARTIFACTS"
         fi
 
-        MANAGER_KIND_CONFIG="${SOURCE_DIR}/multikueue/manager-cluster.kind-${KIND_VERSION}.yaml"
-        if [ !  -f "$MANAGER_KIND_CONFIG" ]; then
-            MANAGER_KIND_CONFIG="${SOURCE_DIR}/multikueue/manager-cluster.kind.yaml"
+	cp "${SOURCE_DIR}/multikueue/manager-cluster.kind.yaml" "$ARTIFACTS"
+
+        #Enable the JobManagedBy feature gates for k8s 1.30+ versions 
+        IFS=. read -r -a varr <<< "$KIND_VERSION"
+        minor=$(( varr[1] ))        
+        if [ "$minor" -ge 30 ]; then
```

Comment:
Can we directly add the feature gate to https://github.com/kubernetes-sigs/kueue/blob/4199c9dd9ce89636eb4e72f5cebb3c9adfba3f0c/hack/multikueue/manager-cluster.kind.yaml instead of this script change?

Because, the featureGate can be specified even if the featureGate has already been graduated to the Beta or GA.
Additionally, AFAIK, the k/k featureGate will be kept even after GA while 2 versions.